### PR TITLE
feat(route): add AIbase

### LINKED ISSUE #16877: https://top.aibase.com
### 类型

社交媒体

### 网站地址

https://top.aibase.com

### 网站描述

AIbase is your all-in-one hub for navigating the exciting world of artificial intelligence. 

### 需要生成什么内容？

产品内容和图像

### 额外描述

_No response_

### 这不是重复的 RSS 请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues) 和 [pull requests](https://github.com/DIYgod/RSSHub/pulls)，以确保该 RSS 尚未被请求。

### PR DESCRIPTION
**Pull request by nczitzk**
## Involved Issue / 该 PR 相关 Issue

Close #16877

## Example for the Proposed Route(s) / 路由地址示例

```routes
/aibase/discover
/aibase/topic/AI
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/aibase/discover.ts [hunk 1]
```diff
@@ -0,0 +1,388 @@
+import { Route } from '@/types';
+
+import { ofetch } from 'ofetch';
```
Comment (TonyRL) [inline lib/routes/aibase/discover.ts:3]:
```suggestion
import ofetch from '@/utils/ofetch';
```

File: lib/routes/aibase/namespace.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import type { Namespace } from '@/types';
+
+export const namespace: Namespace = {
+    name: 'AIBase',
```
Comment (TonyRL) [inline lib/routes/aibase/namespace.ts:4]:
```suggestion
    name: 'AIbase',
```

File: lib/routes/aibase/templates/description.art [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+          {{ set list = strToArray(item.use) }}
+          {{ if list.length === 1 }}
+            {{ list[0] }}
+          {{ else }}
+            {{ each list l }}
+              <li>{{ l }}</li>
+            {{ /each }}
+          {{ /if }}
+        </td>
+      </tr>
```
Comment (TonyRL) [inline lib/routes/aibase/templates/description.art:59]: image

Comment (nczitzk) [reply lib/routes/aibase/templates/description.art:59]: 👌

File: lib/routes/aibase/util.ts [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+/**
+ * Process an array of items to generate a new array of processed items for RSS.
+ * @param items - An array of items to process.
+ * @returns An array of processed items.
+ */
+const processItems = (items: any[]): any[] =>
+    items.map((item) => {
+        const title = item.name;
+        const image = item.imgurl;
+        const description = art(path.join(__dirname, 'templates/description.art'), {
```
Comment (TonyRL) [inline lib/routes/aibase/util.ts:74]: ReferenceError: __dirname is not defined

**Pull request merged**